pkg/image/controller/controller.go

```diff
@@ -163,7 +169,50 @@ func (c *ImportController) Next(stream *api.ImageStream, notifier Notifier) erro
 		if apierrs.IsNotFound(err) && client.IsStatusErrorKind(err, "imageStream") {
 			return ErrNotImportable
 		}
-		glog.V(4).Infof("Import stream %s/%s partial=%t error: %v", stream.Namespace, stream.Name, partial, err)
+		glog.V(4).Infof("Import stream %s/%s partial=%t error: %#+v", stream.Namespace, stream.Name, partial, err)
+		if apierrs.IsInvalid(err) && client.IsStatusErrorKind(err, "ImageStreamImport") {
```

Comment:
@smarterclayton this uglyness is needed to properly report the import errors to users who:

1) Create ImageStream that contains multiple tags (typically examples/imagestreams...)
2) The import fails because the registry used in ImageStream is invalid (for tag or for repository)
3) The error is then logged into glog.V(4) and burried in server log
4) User end up with ImageStream that tells you the tags are being imported... forever.

(in addition, if this happen, all other tags in that IS are not imported even though they are valid... but one problem at the time)...

And since the StatusInvalid will not tell us what tag failed specifically you have to guess it from the error message (which sux big time, but I can't figure out better solution). 

I would said that this code is just a band-aid for having the import error reported properly (since white-listing registries needs to do this), but we should fix the error reporting in general from the image streams in 1.6.